helm_deploy/hmpps-tier/templates/_envs.tpl

```diff
@@ -34,6 +34,15 @@ env:
   - name: ASSESSMENT_ENDPOINT_URL
     value: "{{ .Values.env.ASSESSMENT_ENDPOINT_URL }}"
 
+  - name: CALCULATION_VERSION
```

Comment:
Decoupling the version from these two flags seems a bit risky. We could potentially enable one of the flags and not change the version number, which would potentially invalidate calculations. It certainly introduces more complexity.

I wonder if we could just use the version number for the necessary conditional? Is there anything to say that we can't do Arson and Violence as 1.1 and Ten Months Plus as 1.2 for example

Reply:
I considered this and thought it was too complex having to do `$version >= 1.1` checks and such, but thinking about it again, we'll just remove the feature flag once we're happy so we won't litter the codebase with complex checks.